migrations/v2.js

```diff
@@ -0,0 +1,30 @@
+import {
+  describe,
+  whereContent,
+  whereFromPlugin,
+  mutateContent,
+  checkContent,
+  updatePlugin,
+  getCourse
+} from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-text - v2.0.0 to v2.0.1', async () => {
+  let course;
+  const ariaRegionPath = '_globals._components._text.ariaRegion';
+  whereFromPlugin('adapt-contrib-text - from v2.0.0 to v2.0.1', { name: 'adapt-contrib-text', version: '<2.0.1' });
```

Comment:
Recent change to stop it trying to migrate v1
```suggestion
  whereFromPlugin('adapt-contrib-text - from v2.0.0 to v2.0.1', { name: 'adapt-contrib-text', version: '>=2.0.0 <2.0.1' });
```

Reply:
This would need to be specified on all tasks rather than just the first, so ignoring for now as agreed offline.